docs/metrics/README.md

```diff
@@ -356,11 +356,10 @@ predictable and reliable behavior when excessive cardinality happens, whether it
 was due to a malicious attack or developer making mistakes while writing code.
 
 OpenTelemetry has a default cardinality limit of `2000` per metric. This limit
-can be configured at `MeterProvider` level using the
-`SetMaxMetricPointsPerMetricStream` method, or at individual
-[view](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#view)
-level using `MetricStreamConfiguration.CardinalityLimit`. Refer to this
-[doc](../../docs/metrics/customizing-the-sdk/README.md#changing-maximum-metricpoints-per-metricstream)
+can be configured for individual metrics by using
+[views](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#view)
+and the `MetricStreamConfiguration.CardinalityLimit` setting. Refer to this
+[doc](../../docs/metrics/customizing-the-sdk/README.md#changing-the-cardinality-limit-for-a-metric)
```

Comment:
metric/metrics, metrics should be uncountable here :laughing:?
https://www.wordhippo.com/what-is/the-plural-of/metrics.html#:~:text=The%20noun%20metrics%20is%20uncountable,of%20metrics%20is%20also%20metrics.

Reply:
+1, consider:

_can be configured at individual metric level using the View API and the ..._

Reply:
Updated